wagn/rails/controllers/card_controller.rb

```diff
@@ -44,7 +48,9 @@ def update
 
   def delete
     discard_locations_for card #should be an event
-    params[:success] ||= 'REDIRECT: *previous'
+    if @success.target == card
+      @success.target = :previous
+    end
```

Comment:
I suppose we could put this in an event now...  (though it's already cleaner now than it was)